pkg/statistics/handle/storage/gc.go

```diff
@@ -182,12 +182,12 @@ func ClearOutdatedHistoryStats(sctx sessionctx.Context) error {
 	}
 	count := rows[0].GetInt64(0)
 	if count > 0 {
-		sql = "delete from mysql.stats_meta_history use index (idx_create_time) where create_time <= NOW() - INTERVAL %? SECOND"
+		sql = "batch on create_time limit 10 delete from mysql.stats_meta_history use index (idx_create_time) where create_time <= NOW() - INTERVAL %? SECOND"
```

Comment:
I guess a larger limit is better. We can decide it based on experiments. The NT-DML doc suggests 1,000-1,000,000.

Reply:
It is possible for stats_meta_history to large the limit. but stats_history is too big to happen the OOM.